echopype/clean/api.py

```diff
@@ -40,14 +44,20 @@ def mask_transient_noise(
         Calibrated Sv data with depth data variable.
     func: str, default `nanmean`
         Pooling function used in the pooled Sv aggregation.
-    depth_bin : str, default `20`m
-        Pooling radius along ``depth`` in meters.
-    num_side_pings : int, default `50`
+    depth_bin : str, default `10m`
+        Pooling radius vertically along `depth`.
+    num_side_pings : int, default `25`
         Number of side pings to look at for the pooling.
     exclude_above : Union[int, float], default `250`m
         Exclude all depth above (closer to the surface than) this value.
     transient_noise_threshold : Union[int, float], default `10.0`dB
         Transient noise threshold value (dB) for the pooling comparison.
```

Comment:
Let's change this to do "10.0dB" (also allow variations such as "10.0 dB") so that we force people to think about the units.

Reply:
I couldn't comment on the exclude_above part, since it was not part of the changed code, but could you also do the same for `exclude_above`? like `250`m to "250m" ?

Reply:
Yeah good idea, I'll add dB. Also, see my comment on backticks vs double quotes above.